lstchain/scripts/lstchain_merge_hdf5_files.py

```diff
@@ -72,6 +72,11 @@
     help='Skip checks when merging files'
 )
 
+parser.add_argument(
+    '--keys-to-copy',
+    nargs="+", default=[''],
+    help='List of duplicated keys to be copied and not to be merged'
```

Comment:
I don't think this should be a command line argument. We know which keys are event-wise and should be merged and which are global so shouldn't be merged.

Let's define this in the code.

Reply:
Indeed, this would be the best, but since this seems a general merging function I was not sure we wanted to specify it inside the function. If we agree on that I prefer it also.

Reply:
It's a general merging function for our data, we do know all possible keys.

Reply:
I'm reopening this conversation because I think it is not resolved yet. Having this command line argument makes it a bit confusing at the same time as having a set of default keys to be copied. Do we really need this additional option? I think they can be simply defined in lstchain/io/io.py